skypy/galaxy/_spectrum_loaders.py

```diff
@@ -94,37 +101,46 @@ def skypy_data_loader(module, name, *tags):
     return spectra
 
 
-def decam_loader(*bands):
-    '''load DECam bandpass filters'''
+def speclite_loader(name, *bands):
+    '''load data from the speclite package'''
+
+    # result is spectrum or list of spectra
+    spectra = None
+
+    # load each band sperately
+    for band in bands:
+
+        # get resource filename from name and bands
+        filter_name = f'{name}-{band}'
 
-    # download DECam filter data
-    filename = download_file(
-            'http://www.ctio.noao.edu/noao/sites/default/files/DECam/STD_BANDPASSES_DR1.fits')
+        # load the filter response as a speclite.FilterResponse object
+        filter_object = speclite.filters.load_filter(filter_name)
 
-    # load the data file
-    data = astropy.table.Table.read(filename, format='fits')
+        # get the spectral axis as astropy quantity
+        spectral_axis = units.Quantity(filter_object._wavelength, units.angstrom)
```

Comment:
speclite internally converts to its `default_wavelength_unit` — also, don't use the private `._wavelength` member, use the `.wavelength` accessor

```suggestion
        spectral_axis = filter_object.wavelength*speclite.filters.default_wavelength_unit
```